### DISCUSSION TITLE
jotai.org website scroll is broken on Mac

### DISCUSSION THREAD

**Discussion by keelangibb** [pmndrs/jotai#2425] [Bug report] [answered]
I am on the homepage and I noticed that the left scroll bar does not work on Mac. When you hover over the left section and try to scroll with the mouse pad (2 finger scroll) it scrolls the whole page and not the left scroll bar. This makes it hard to navigate the documentation because you have to click the left bar and then drag it down. I also tried with a normal mouse and it is the same result. This works fine on windows computers.

**Comment by DavidJFelix** [comment]
I'm not seeing this in chrome 121 on mac. What browser are you using and what scroll setting do you have in Preferences > Appearance 
<img width="478" alt="image" src="https://github.com/pmndrs/jotai/assets/263432/d816ce7e-6a0a-4151-bee3-489942956644">

**Reply by keelangibb** [reply]
Safari Version 17.2.1 (19617.1.17.11.12)
It is set to automatically

**Reply by keelangibb** [reply]
So it looks like it works on chrome but not on safari on my computer.

**Comment by rothsandro** [comment, marked as answer]
PR #2426 should fix it
